build/includes/sdk.mk

```diff
@@ -180,6 +180,16 @@ run-sdk-conformance-test-rust:
 	# run with feature flags enabled and with RUN_ASYNC=true
 	DOCKER_RUN_ARGS="$(DOCKER_RUN_ARGS) -e RUN_ASYNC=true" $(MAKE) run-sdk-conformance-test SDK_FOLDER=rust GRPC_PORT=9004 HTTP_PORT=9104 FEATURE_GATES=PlayerTracking=true TESTS=$(DEFAULT_CONFORMANCE_TESTS),$(ALPHA_CONFORMANCE_TESTS)
 
+run-sdk-conformance-test-csharp:
+	# run without feature flags
+	$(MAKE) run-sdk-conformance-test SDK_FOLDER=csharp GRPC_PORT=9005 HTTP_PORT=9105
+	# run without feature flags and with RUN_ASYNC=true
+	DOCKER_RUN_ARGS="$(DOCKER_RUN_ARGS) -e RUN_ASYNC=true" $(MAKE) run-sdk-conformance-test SDK_FOLDER=csharp GRPC_PORT=9005 HTTP_PORT=9105
+	# run with feature flags enabled
+	$(MAKE) run-sdk-conformance-test SDK_FOLDER=csharp GRPC_PORT=9005 HTTP_PORT=9105 FEATURE_GATES=PlayerTracking=true TESTS=$(DEFAULT_CONFORMANCE_TESTS),$(ALPHA_CONFORMANCE_TESTS)
+	# run with feature flags enabled and with RUN_ASYNC=true
```

Comment:
You don't need `RUN_ASYNC` -- it was Rust specific.

You can likely copy from `run-sdk-conformance-test-go` instead.